plugins/zenoh-plugin-storage-manager/src/replica/align_queryable.rs

```diff
@@ -20,6 +20,8 @@ use std::str;
 use std::str::FromStr;
 use zenoh::payload::StringOrBase64;
 use zenoh::prelude::r#async::*;
+use zenoh::sample_builder::TimestampBuilderTrait;
+use zenoh::sample_builder::ValueBuilderTrait;
```

Comment:
Similar. Traits should be part of the prelude.

Reply:
done